docs/adoptability.md

```diff
@@ -913,8 +917,99 @@ gaps or useful opportunties?
 
 ## Evaluation Checklist
 
-(TODO: This will be a summarized, curated checklist based on the
-sections above, particularly the HOWTO items in them, and will refer
-back to them.  Because this will be based on the material above, we'll
-wait until the module has been through some review and settles down a
-bit more before making this summarized checklist.)
+This section summarizes the previous considerations around adoptability in a 
+simple, unprioritized checklist.
+ 
+**Does the product meet your functional requirements?**
+ 
+- Do the functional Application Programming Interfaces (APIs) you’ll need exist, 
+and are they well documented? Can you find examples of these APIs being used in 
+ways similar to your plans?
+- Will your data be portable? 
+    - Can you extract or import non-PII data in a non-proprietary format? (a 
+    requirement of the DPG standard!)
+    - Can you import and export data in the formats you expect to need? 
+    - Can you generally import and export data with an acceptable level of pre- 
+    and post- action cleaning and manipulation? 
+- Is the product extensible? 
+    - Do many extensions exist, and are they being used? 
+    - If you are evaluating vendors to help you with an extensible product, has 
+    the vendor created successful extensions? 
```

Comment:
For clarity, might rephrase as "does the vendor have a history of creating successful extensions for others?".

Reply:
This is now addressed in PR #72.